tgui/packages/tgui/interfaces/ChemHeater.js

```diff
@@ -0,0 +1,130 @@
+import { useBackend } from "../backend";
+import { AnimatedNumber, Box, Button, Dimmer, Flex, Icon, Knob, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { getTemperatureColor, getTemperatureIcon, getTemperatureChange } from './common/temperatureUtils.js';
+import { NoContainer, ReagentGraph, ReagentList } from './common/ReagentInfo.js';
+
+export const ChemHeater = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { containerData, isActive, targetTemperature } = data;
+
+  return (
+    <Window
+      title="Reagent Heater/Cooler"
+      width={320}
+      height={385}>
+      <Window.Content>
+        <ChemDisplay container={containerData} targetTemperature={targetTemperature} active={isActive} />
+        <Section title="Temperature Control">
+
+          <Stack align="center">
+            <Stack.Item>
+              <Knob
+                animated
+                size={2}
+                value={targetTemperature}
+                minValue={0}
+                maxValue={1000}
+                format={value => value + " K"}
+                onDrag={(e, value) => act('adjustTemp', { temperature: value })}
+              />
+            </Stack.Item>
+            <Stack.Item grow overflow="hidden" py={1}>
```

Comment:
`Stack.Item`s should have sensible margin by default. Not sure exactly what the default is, but worth seeing if this padding is needed at all.

(Possible, tricky to review code and visualize at once.)